uber/templates/signups/shirt_size.html

```diff
@@ -3,18 +3,56 @@
+        We ask that all staff wear their staff shirt whenever they are on shift so that they are easily identifiable at a
+        distance by attendees, security, and fellow staff. While wearing the staff shirt, people may be more likely to ask
+        you for assistance, but don't panic! We'll provide a map and schedule for you to carry, which should answer most
+        questions, and we recommend downloading the event Guidebook app as well. When in doubt, the only two things you
+        really need to know are the location of the info desk and the location of security, so that you can direct the
+        person to someone who can provide more assistance.
+    </p>
+    <p><b>When you are off-shift, please do not wear the staff shirt unless you are prepared to provide assistance.</b></p>
+    <p><i>
+        PRO-TIP: Wearing a long sleeve thermal shirt beneath the staff shirt can help keep you warm, keep the staff
+        shirt fresh through the event, and make it easier to take off the staff shirt at the end of your shift.
+    </i></p>
+    <p>
+        Staff shirts help make the event a more safe and welcome place to be, and while we strongly prefer you to wear
+        a staff shirt while on shift, you are not required to do so. We understand that you may not be comfortable with
+        people approaching you, or that your shift may have a level of sensitivity which requires that you not be obviously
+        identifiable as staff. If you strongly dislike t-shirts, check with your department head to see if they can provide
+        you with a visibility vest while on shift. 
+    </p>
+    <p>
+        To prevent confusion, please do not wear a previous year's staff shirt at this event, whether or not you are on shift.
+        <b>If you are wearing an old staff shirt, you may be asked to change.</b>
+    </p>
+    <p>
+        As staff, you receive two complimentary shirts, and since you'll be potentially wearing the staff shirt a substantial
+        portion of the event, we give you the option for both of them to be the staff shirt design.
+        {% if c.AFTER_SHIRT_DEADLINE %}
+            <b>Because we are past the deadline, your choice has been locked in.</b>
+        {% endif %}
+        <br/> <input type="radio" name="second_shirt" value="{{ c.TWO_STAFF_SHIRTS }}" {% if attendee.second_shirt == c.TWO_STAFF_SHIRTS %}checked{% endif %} {% if c.AFTER_SHIRT_DEADLINE %}disabled{% endif %} /> I'd like {{ c.SHIRTS_PER_STAFFER }} staff shirts, to help keep them fresh through the event.
+        <br/> <input type="radio" name="second_shirt" value="{{ c.STAFF_AND_EVENT_SHIRT }}" {% if attendee.second_shirt != c.TWO_STAFF_SHIRTS %}checked{% endif %} {% if c.AFTER_SHIRT_DEADLINE %}disabled{% endif %} /> I'd like {{ c.SHIRTS_PER_STAFFER - 1 }} staff shirt{{ (c.SHIRTS_PER_STAFFER - 1)|pluralize }}, and 1 standard event-themed swag shirt. <i>I can keep a staff shirt fresh through the event.</i>
```

Comment:
Normally I go for a dropdown for this kind of thing, but given the amount of verbiage it seemed better to use radio buttons.  Since there were only two options with a little bit of extra weird logic (to account for the pseudo-default value and the deadline-disabling) I did this by hand with some inline if-statements rather than writing or using a macro.